clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -459,18 +468,16 @@ bool PointerVariableConstraint::emitArraySize(std::ostringstream &Pss,
     OriginalArrType Oat = i->second.first;
     uint64_t Oas = i->second.second;
 
-    if (EmitName == false) {
-      EmitName = true;
-      Pss << getName();
-    }
+    std::ostringstream SizeStr;
 
     switch (Oat) {
       case O_SizedArray:
         if (!EmittedCheckedAnnotation) {
-          Pss << (Nt ? " _Nt_checked" : " _Checked");
-          EmittedCheckedAnnotation = true;
+          SizeStr << (Nt ? " _Nt_checked" : " _Checked");
+          //EmittedCheckedAnnotation = true;
```

Comment:
If this line can be commented out, I think we should be able to remove `EmittedCheckedAnnotation` entirely. It's initialized to `false` and this is the only other place it's written. So, two questions: should that line be removed and, if yes, can we remove `EmittedCheckedAnnotations` everywhere.

Reply:
So, the line *is* removed; you are saying that more removal can be done, as a consequence?